docs/operate/get-started/supported-hardware.md

```diff
@@ -5,5 +5,367 @@ weight: 20
+
+{{% alert title="Info" color="tip" %}}
+Some components are supported by drivers built into `viam-server`.
+The list above contains both modular components and built-in components.
+You can also browse these same built-in and modular registry components on your machine's configuration page in the Viam app.
+
+The Viam Registry is the storage and distribution system for not just hardware modules but also software modules (called services), ML models, and ML model training scripts.
+You can browse the [Viam Registry in the Viam app](https://app.viam.com/registry?type=Module).
+{{% /alert %}}
+
+### For use with microcontrollers
+
+The following is a selection of components (some built-ins and some modules) written for use with `viam-micro-server`.
+To use any of the built-in components, configure them according to their readmes.
+To use a module with `viam-micro-server`, you need to [build firmware that combines `viam-micro-server` with one or more modules](/operate/get-started/other-hardware/micro-module).
+
+<!--prettier-ignore-->
+| Model | Description | Built-in |
+| ----- | ----------- | -------- |
+| `gpio` | A servo controlled by GPIO pins. [Configuration info](/components/servo/gpio-micro-rdk/). | Yes |
+| `two_wheeled_base` | A robotic base with differential steering. [Configuration info](/components/base/two_wheeled_base/). | Yes |
+| `free_heap_sensor` | Ships with `viam-micro-server`. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | No |
+| `wifi_rssi_sensor` | Ships with `viam-micro-server`. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | No |
+| `moisture_sensor` | [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | No |
+| `water_pump` | [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | No |
+
+## Configure hardware on your machine
+
+After installing `viam-server` or `viam-micro-server` on your computer or microcontroller, you can configure hardware components on your machine's page in the [Viam app](https://app.viam.com):
+
+1. Click the **+** button on your machine's **CONFIGURE** tab.
+1. Click **Component**, then select from available components from the Viam Registry (as well as built-in resources).
+1. Use the link to the module README to find information on configuring that specific component.
+
+   {{<gif webm_src="/integrate/configure.webm" mp4_src="/integrate/configure.mp4" alt="Configuring a board and ultrasonic sensor." max-width="600px">}}
+
+   When you add a modular resource _from the registry_, the module that provides it is automatically added at the same time, generating a configuration card for the modular resource and a separate one for the module.
+   If you add a built-in component, there will only be a configuration card for the component.
+
+### Modular resource configuration details
```

Comment:
I wonder whether this and the module config details should stay in reference content. I think most of the time this info here (and the info for module config) will only be used as reference.

Reply:
Agreed. Will move.